### PR TITLE
created type definitions for "geometric" package

### PR DESCRIPTION
**Pull request by lindapaiste**
type definitions for https://www.npmjs.com/package/geometric

### CODE REVIEW

File: types/geometric/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,198 @@
+// Type definitions for geometric 2.2
+// Project: https://github.com/HarryStevens/geometric#readme
+// Definitions by: Linda Paiste <https://github.com/DefinitelyTyped>
```
Comment (peterblazejewicz) [inline types/geometric/index.d.ts:3]:
```suggestion
// Definitions by: Linda Paiste <https://github.com/lindapaiste>
```
unless you want to stay off-the grid as the owner

File: types/geometric/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
```
Comment (peterblazejewicz) [inline types/geometric/index.d.ts:5]:
```suggestion

export as namesapce geometric;

```

File: types/geometric/index.d.ts (hunk 1 above)
Comment (lindapaiste) [reply types/geometric/index.d.ts]: I'm fine attaching myself to it.  That was just an oversight.  Thanks for catching it!

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@lindapaiste thx for the contribution!

**Review by andrewbranch — APPROVED** [review]

**Comment by lindapaiste** [comment]
Ready to merge

**Pull request merged**